IFTB.bs

```diff
@@ -0,0 +1,535 @@
+If the font contains a gvar table it must come immediately before the glyf table.
+
+There are no other ordering requirements. However, it is highly recommended that the IFTB and cmap
+tables come early in the list and before any other lengthy table.
+
+IFTB Font Formats {#iftb-font-formats}
+--------------------------------------
+
+The specification allows for two initial font formats:
+
+1.   The uncompressed format is a slight variant of the OpenType "sfnt" format, but with a version tag
+     of <code>IFTB</code> instead of <code>0x00010000</code> or <code>OTTO</code>
+2.   The compressed format is a WOFF2 encoding of the uncompressed format with all tables in the same
+     order.  The sfnt "flavor" in the WOFF2 header must accordingly also be <code>IFTB</code>
+
+It is expected that the client will almost always download a WOFF2-encoded file, which it will then
+unpack into the uncompressed format and modify as needed.
+
+Before loading a copy of the uncompressed font into the browser, or making it available to some other
+renderer, the version should be changed to <code>0x00010000</code> if the font has a glyf table or
+<code>OTTO</code> if it has a CFF or CFF2 table.  However, if the file is cached it should be stored
+with the <code>IFTB</code> version.  (The differing version is to help prevent an IFTB-encoded font
+from being used accidentally as a standard font, which will generally result in blank spaces where some
+glyphs should be rendered.)
+
+IFTB Table {#iftb-table}
+------------------------
+
+The IFTB table contains information needed to determine what chunks are needed for a given font subset
+description, and then to retrieve and integrate those chunks. It is similar in layout to analogous
+OpenType tables, and is accordingly described in terms of the [OpenType Specification Data
+Types](https://learn.microsoft.com/en-us/typography/opentype/spec/otff#data-types). As with OpenType
+all fields use Big Endian byte orderng.
+
+### IFTB Table Header ### {#iftb-table-header}
+
+```
+uint16     majorVersion   - set to 1
+uint16     minorVersion   - set to 0
+uint32     reserved       - not used, set to 0
+uint32     id[4]          - ID uniquely identifying this IFTB font
+uint16     flags          - not yet used
+uint32     chunkCount     - number of bins in this IFTB font
+uint32     glyphCount     - must match field in maxp table
+Offset32   CFFCharStringsOffset  - 0 if glyf-based
+Offset32   gidMapOffset
+Offset32   chunkOffsetListOffset
+Offset32   featureMapOffset
+```
+
+### IFTB Chunk Set ### {#iftb-chunk-set}
+
+The header is followed by the chunk set:
+
+```
+uint8     chunkSet[(chunkCount + 7) / 8]
+```
+
+The chunk set is a bit array indicating whether the corresponding chunk is
+present.  The bits for chunks 0 through 7 are in chunkSet[0], those for 8
+through 15 are in chunkSet[1], and so on.  Within a byte the lowest chunk index
+is represented by the 1s bit, then the 2s, then the 4s, and so on.
```

Comment:
To be as unambiguous as possible it's helpful to define the mapping to bits within a byte using the most and least significant bit (eg. the least significant bit is chunk 0, the most signifcant bit is chunk 7)